repo/packages/W/weavescope/0/resource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/errordeveloper/universe/version-2.x/repo/packages/W/weavescope/0/weave-logo-transparent-small.png",
+    "icon-medium": "https://raw.githubusercontent.com/errordeveloper/universe/version-2.x/repo/packages/W/weavescope/0/weave-logo-transparent-medium.png",
+    "icon-large": "https://raw.githubusercontent.com/errordeveloper/universe/version-2.x/repo/packages/W/weavescope/0/weave-logo-transparent-large.png",
```

Comment:
we don't support the images coming from github.  all resources that are not docker should be backed by a CDN